src/client/Microsoft.Identity.Client/WwwAuthenticateParameters.cs

```diff
@@ -50,6 +58,17 @@ public class WwwAuthenticateParameters
         /// </summary>
         public string Error { get; set; }
 
+        /// <summary>
+        /// AuthScheme.
+        /// See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/WWW-Authenticate#syntax for more details
+        /// </summary>
+        public string AuthScheme { get; private set; }
+
+        /// <summary>
+        /// Pop Nonce. This is acquired from with the POP WWW-Authenticate header.
```

Comment:
This is not true - it is acquired from any type of header which has the "nonce" param.